scripts/gen-edpm-kustomize.sh

```diff
@@ -31,6 +31,30 @@ if [ -z "$DEPLOY_DIR" ]; then
     echo "Please set DEPLOY_DIR"; exit 1
```

Comment:
The changes looks good.  Based on the commit message, this file change is related to above commit message.
Can we move Makefile changes to a separate pr?

Reply:
actually, the Makefile changes are related, if you use `make openstack_wait_deploy` it will ensure the variables from `make edpm_deploy` won't be empty

Reply:
Thank you for clarifying it. Let me use these targets in ci-framework side.